app/views/errors/internal_server_error.html.haml

```diff
@@ -0,0 +1,11 @@
+%html
```

Comment:
Are we using the same layout as the application layout? If so, we don't need the html tags here. If not, you need to include the partials from the application layout to render the same header and footer.